zng/docs/spec.md

```diff
@@ -468,17 +467,17 @@ A TZNG type binding has the following form:
 #<type-tag>:<type-string>
 ```
 Here, `<type-tag>` is a string decimal integer and `<type-string>`
-is a string defining a record type (`<rtype>`) according to the [TZNG type grammar](#42-type-grammar). They create
+is a string defining a type (`<type>`) according to the [TZNG type grammar](#42-type-grammar). They create
 a binding between the indicated tag and the indicated type.
 
 ### 4.1.2 Type Alias
 
 A TZNG type alias has the following form:
 ```
-#<type-name>:<type-string>
+#<type-name>=<type-string>
```

Comment:
I'm shocked I let this one elude me for so long, but it looks to me like the spec was dead wrong before:

```
$ zq -t conn.log.gz | head -2
#zenum=string
#0:record[_path:string,ts:time,uid:bstring,id:record[orig_h:ip,orig_p:port,resp_h:ip,resp_p:port],proto:zenum,service:bstring,duration:duration,orig_bytes:uint64,resp_bytes:uint64,conn_state:bstring,local_orig:bool,local_resp:bool,missed_bytes:uint64,history:bstring,orig_pkts:uint64,orig_ip_bytes:uint64,resp_pkts:uint64,resp_ip_bytes:uint64,tunnel_parents:set[bstring]]
```